src/dsp_tools/commands/excel2json/new_lists/utils.py

```diff
@@ -32,9 +56,9 @@ def get_all_languages_for_columns(columns: pd.Index[str], ending: str = r"(\d+|l
     return set(res for x in columns if (res := get_lang_string_from_column_name(x, ending)))
 
 
-def get_preferred_language(columns: pd.Index[str], ending: str = r"(\d+|list)") -> str:
+def from_columns_get_preferred_language(columns: pd.Index[str], ending: str = r"(\d+|list)") -> str:
```

Comment:
The naming seems a bit unusual, I would prefer `get_preferred_language_from_columns()`